### PR TITLE
New code for the highscore task

### PR DESCRIPTION
**Pull request by gabe2762**
database updates with the score, name, and game_id. I am not attaching the CLI code that I changed because I do not want to cause A merge conflict, so I will post the file in roughdraft code. The CLI giveup function currently prints out what should be getting sent to the database only as a test to make sure it is actually sending the right info, it will not be a permanent part of the code.

### CODE REVIEW

**Comment by gabe2762** [comment]
Has the documentation that Austin wanted in the last pr, I ran into some conflicts with my own .db file but couldn't open it in vs code to resolve the conflicts so I just wiped and recloned the repo and added my changes back in (not the best strat but its one that works) So sorry for the new pr instead of sticking with the old one

File: MVC/database/connectDB.py [hunk 1]
```diff
@@ -12,6 +12,10 @@
 abs_path2 = os.path.abspath(db_dir2)
 #print(abs_path2)
 
+check_dir3 = os.path.dirname(os.path.abspath(__file__))
+db_dir3 = os.path.join(check_dir3,".","highscores.db")
+abs_path3 = os.path.abspath(db_dir3)
+#print(abs_path3)
```
Comment (tmhughes1) [inline MVC/database/connectDB.py:18]: remove this

File: MVC/model/Model.py [hunk 2]
```diff
@@ -128,6 +142,11 @@ def NewPuzzleBase(self,userInput):
         self.p1.gaUserLetters, self.p1.gaReqLetter = np.baseGame(userInput)
         self.p1.getList = wl.generateWordList(self.p1.gaReqLetter, self.p1.gaUserLetters)
         self.calculateTotalPoints(self.p1.getList)
+        self.game_id = self.generate_game_id()
+    
+    #getter function to get the game_id
+    def get_game_id(self):
```
Comment (tmhughes1) [inline MVC/model/Model.py:148]: don't think this should be here

**Review by tmhughes1 — CHANGES_REQUESTED** [review]
The commenting of methods and naming (ex get_game_id instead of getGameID) are not consistent with the rest of our code base. Also there is commented out code that does not need to be left in. I also commented on the getter function within the model as I don't think it should be there.

**Pull request closed without merging**